src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockType.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use ForkCMS\Core\Domain\Form\EditorType;
+use ForkCMS\Core\Domain\Form\TitleType;
+use ForkCMS\Modules\Backend\Domain\User\User;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ContentBlockType extends AbstractType
+{
+    public function __construct(
+        private readonly Security $security
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        /** @var ContentBlockDataTransferObject $data */
+        $data = $builder->getData();
+
+        /** @var User $user */
+        $user = $this->security->getUser();
+
+        $data->userId = $user->getId();
+
+        $builder->add(
+            'title',
+            TitleType::class
+        )->add(
+            'text',
+            EditorType::class,
+            [
+                'required' => true,
+                'label' => 'lbl.Content',
+            ]
+        );
+
+        /*$templates = $this->getPossibleTemplates($options['theme']);
+        // if we have multiple templates, add a dropdown to select them
+        if (count($templates) > 1) {
+            $builder->add(
+                'template',
+                ChoiceType::class,
+                [
+                    'required' => true,
+                    'label' => 'lbl.Template',
+                    'choices' => $templates,
+                    'choice_translation_domain' => false,
+                ]
+            );
+        }*/
+
+        $isVisibleOptions = [
+            'label' => 'lbl.VisibleOnSite',
+            'required' => false,
+        ];
+
+        if (!array_key_exists('data', $options)) {
+            $isVisibleOptions['attr']['checked'] = 'checked';
+        }
+
+        $builder->add(
+            'isVisible',
+            CheckboxType::class,
+            $isVisibleOptions
+        );
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefault('data_class', ContentBlockDataTransferObject::class);
+    }
+
+    /*private function getPossibleTemplates(string $theme): array
```

Comment:
Why didn't you implement this?

Reply:
tbh I'm not sure why a ContentBlock needs an editable template-path. What does this do?

Reply:
@carakas I don't think we have ever used this? I don't think this is really needed?